src/shared/components/msktable/MSKTable.tsx

```diff
@@ -20,13 +21,14 @@ export type Column<T> = {
     render:(data:T)=>JSX.Element;
     download?:(data:T)=>string;
     tooltip?:JSX.Element;
+    firstSortDirection?:SortDirection;
```

Comment:
feels a little confusing, initial feels like it implies it'll be like that on start-up